src/helper.js

```diff
@@ -113,15 +114,19 @@ export const calculateExchangeRate = (satoshis, settings) => {
  * @param  {Object} settings Contains the current exchange rate
  * @return {string}          The corresponding value label
  */
-export const toAmountLabel = (satoshis, settings) => {
+export const toAmountLabel = (satoshis, settings, centTenths = false) => {
   if (
     !Number.isInteger(satoshis) ||
     typeof settings.displayFiat !== 'boolean'
   ) {
     throw new Error('Invalid input!');
   }
   return settings.displayFiat
-    ? formatFiat(calculateExchangeRate(satoshis, settings), settings.fiat)
+    ? formatFiat(
+        calculateExchangeRate(satoshis, settings),
+        settings.fiat,
+        centTenths
+      )
```

Comment:
Couldn't we just change `formatFiat` to prepend a `< ` if the string matches `/0[,.]00/`?

Reply:
Do you mean if `formatFiat` is going to return `0.00`, it will instead return `<0.00`? I think I'm misunderstanding...

Reply:
Exactly. Why do we need to add an arg to `toAmountLabel`?